SCons/Node/Python.py

```diff
@@ -111,7 +118,7 @@ def get_text_contents(self) -> str:
         cannot use the actual node.built_value."""
         ###TODO: something reasonable about universal newlines
         contents = str(self.value)
-        for kid in self.children(None):
+        for kid in self.children(False):
```

Comment:
Just so it gets a vote, I've been making this change, slowly, in places where it makes sense (which means @bdbaddog has approved some of them ;) ): (a) if it's really a bool, where we only actually care about truthiness, don't use `None`  as the false value, actually use `False`. (b) for strings, if an empty string is good enough to indicate "not set", then use that, instead of `None`.

In this case I'd actually use the kwarg: `...in self.children(scan=False)` to be more clear (yeah, it's _only_ a test...)